stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Extensions.kt

```diff
@@ -30,6 +31,10 @@ fun Fragment.showToast(@StringRes resId: Int) {
     showToast(getString(resId))
 }
 
+fun Context.appThemeContext(): Context {
+    return ContextThemeWrapper(this, R.style.AppTheme)
+}
+
```

Comment:
How about making property instead of the function as the below?

```kotlin
val Context.appThemeContext : Context
    inline get() = ContextThemeWrapper(this, R.style.AppTheme)
```

Or we can consider an extension of `LayoutInflater` such as `appThemeLayoutInflater` for removing all boilerplates.

```kotlin
val Context.appThemeLayoutInflater: LayoutInflater
    inline get() = LayoutInflater.from(ContextThemeWrapper(this, R.style.AppTheme))
```

Reply:
I like the suggestion above to make it a property, seems like it would be a nice approach.

The inflater extension I personally would like to avoid, as an inflater can (in some places) require a parent view, sometimes it's null, etc. 

I'd also love some docs around this, if possible, would love more comments all over the call sites, but that's a lot of work and maybe we can do without it for now.